piet-gpu/shader/coarse.comp

```diff
@@ -146,6 +151,10 @@ void main() {
     uint part_start_ix = 0;
     uint ready_ix = 0;
 
+    uint drawmonoid_start = conf.drawmonoid_alloc.offset >> 2;
+    uint drawtag_start = conf.drawtag_offset >> 2;
+    uint drawdata_start = conf.drawdata_offset >> 2;
+    uint drawinfo_start = conf.drawinfo_alloc.offset >> 2;
```

Comment:
Is it worth storing these as indices rather than byte offsets? I don't think this is a perf concern, but it might improve clarity if all memory references are at 32-bit granularity.

Reply:
This is a good question and worth reconsidering. Basically this code started in Metal where pointers are byte indices, and more or less the same is true in HLSL. Similarly offsets for CPU-side encoding are more naturally at byte granularity. But I agree this might be worth reconsidering now that we're writing more low-level GLSL.